metaflow/plugins/argo/argo_workflows_deployer.py

```diff
@@ -188,6 +225,59 @@ def delete(instance: DeployedFlow, **kwargs):
     return command_obj.process.returncode == 0
 
 
+def from_deployment(identifier: str, metadata: str = None):
+    client = ArgoClient(namespace=KUBERNETES_NAMESPACE)
+    workflow_template = client.get_workflow_template(identifier)
+
+    metadata_annotations = workflow_template.get("metadata", {}).get("annotations", {})
+
+    flow_name = metadata_annotations.get("metaflow/flow_name", "")
+    parameters = json.loads(metadata_annotations.get("metaflow/parameters", {}))
```

Comment:
can you verify all kinds of parameters, including includefile work as expected?

Reply:
JSON and includefile work now..